### ISSUE TITLE
max_val of amplitude SSIM and PSNR loss

### ISSUE THREAD

**Issue by ytkend** [liangs111/tensor_holography#14]
Hello @liangs111 ! I have some questions about the following codes in `main.py`:

```python
ssim_amp_loss = tf.reduce_mean(tf.image.ssim(tf.transpose(y_gt_amp, [0,2,3,1]), tf.transpose(y_out_amp, [0,2,3,1]), 1.0))
psnr_amp_loss = tf.reduce_mean(tf.image.psnr(tf.transpose(y_gt_amp, [0,2,3,1]), tf.transpose(y_out_amp, [0,2,3,1]), 1.0))
```

I think the amplitude outputs are normalized to [0, sqrt(2)], but the `max_val` of `tf.image.ssim` and `tf.image.psnr` are set to 1. Is the correct `max_val` sqrt(2)?

Also, is there any reason for normalizing the amplitude outputs to [0, sqrt(2)] instead of [0, 1]?
